### DISCUSSION TITLE
How to add unmapped property to a row that uses [GenerateFields]?

### DISCUSSION THREAD

**Discussion by shakazulu89** [serenity-is/Serenity#7167] [Help]
Hey

Lets say i want to add a property to a row, for example "IsDirty", its unmapped and not used in any columns, forms, dialogs, etc.. I used to be able to just add:
`public string IsDirty{ get; set; }`

but now this throws an error says it lacks getter and setter. This is because the row is using [Generatefields] attribute.

im just wondering whats the proper way to add such a field? should i just add it like any other field:
`[NotMapped]
public string IsDirty
{
    get => fields.IsDirty[this]; set => fields.IsDirty[this] = value;
}`

or is there another way i should add such a field?

Thanks!
Roy

**Comment by VictorTomaili** [comment]
GenerateFields is pro feature, you need to ask it on StartSharp repository.

**Comment by shakazulu89** [comment]
oh, sorry didnt know! thanks
